src/patternfly/components/ChipGroup/chip-group.scss

```diff
@@ -19,7 +19,7 @@
   --pf-c-chip-group__label--PaddingBottom: var(--pf-global--spacer--xs);
   --pf-c-chip-group__label--PaddingLeft: 0;
   --pf-c-chip-group__label--FontSize: var(--pf-global--FontSize--sm);
-  --pf-c-chip-group__label--Maxwidth: #{pf-size-prem(120px)};
+  --pf-c-chip-group__label--MaxWidth: #{pf-size-prem(120px)};
```

Comment:
changing the case [breaks the var](https://codepen.io/mcoker/pen/ExxWMra) :( Can you leave a comment or create an issue to fix this in a breaking change release?